src/editors/query/query.url.tsx

```diff
@@ -30,11 +30,14 @@ export const Method = ({ query, onChange, onRunQuery }: { query: InfinityQuery;
   if (query.source === 'inline' || query.source === 'azure-blob') {
     return <></>;
   }
-  const URL_METHODS: SelectableValue[] = [
+  const URL_METHODS: Array<SelectableValue<InfinityURLMethod>> = [
     { label: 'GET', value: 'GET' },
     { label: 'POST', value: 'POST' },
+    { label: 'PUT *', value: 'PUT' },
+    { label: 'PATCH *', value: 'PATCH' },
+    { label: 'DELETE *', value: 'DELETE' },
```

Comment:
I would remove `*` because we are not adding any additional explanation. Also, what do you think about hiding them and only showing them is they are enabled?